### ISSUE TITLE
Category Selection

### ISSUE THREAD

**Issue by everytimeis80** [DongshuoYin/garbage_dump_detection#2]
I found that the types of annotations in the data set used in this paper are as follows.'disposed garbage', 'domestic garbage', 'agriculture forestry', 'construction waste'.Could you please tell me that it is equivalent to the following labels mentioned in the paper?"Covered Waste",'Domestic Waste','Agricultural Waste','Construction Waste'.Thanks for reading this question.

**Comment by DongshuoYin** [comment]
Sure, Disposed Garbage is Covered Waste and Agriculture Forestry is Agricultural Waste.











Yin

---Original---
From: ***@***.***&gt;
Date: Thu, Aug 3, 2023 10:35 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [DongshuoYin/garbage_dump_detection] Category Selection (Issue #2)




 
I found that the types of annotations in the data set used in this paper are as follows.'disposed garbage', 'domestic garbage', 'agriculture forestry', 'construction waste'.Could you please tell me that it is equivalent to the following labels mentioned in the paper?"Covered Waste",'Domestic Waste','Agricultural Waste','Construction Waste'.Thanks for reading this question.

**Comment by everytimeis80** [comment]
Thank you for answering my question, but I also found that there are "industry waste" and "mining waste" in the training set. May I ask what these two examples mean?

**Comment by DongshuoYin** [comment]
These categories can be ignored.









Yin

---Original---
From: ***@***.***&gt;
Date: Thu, Aug 3, 2023 11:34 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DongshuoYin/garbage_dump_detection] Category Selection (Issue#2)




 
Thank you for answering my question, but I also found that there are "industry waste" and "mining waste" in the training set. May I ask what these two examples mean?

**Comment by everytimeis80** [comment]
Thank you for your reply
